my_list.rb

```diff
@@ -0,0 +1,20 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(*list)
+    @list = list
+  end
+
+  def each(&block)
+    @list.each(&block)
+  end
```

Comment:
- [ ] You should create you're own `each` method instead of using the ruby method

Reply:
Done.

Reply:
Good job so far 👍🏼  , the idea here is to implement your own basic each method and make it execute a block on each element ==>

````ruby
def each
    return to_enum(:each) unless block_given?
    counter = 0
    while counter < @list.length
      yield(@list[counter])
      counter += 1
    end
    @list
end
````

Reply:
Implemented as instructed.